modules/_labs/header/react/README.md

```diff
@@ -384,3 +360,143 @@ Default: `HeaderTheme.White`
 > The title displayed on the header next to the logo.
 
 Default `''`
+
+# Search Bar
+
+_Intended to be used in conjunction with the `Header` component_
+
+A component that contains a search bar with autocomplete functionality.
+
+## Usage
+
+```tsx
+import {SearchBar} from '@workday/canvas-kit-labs-react-header';
+import {MenuItem} from '@workday/canvas-kit-labs-react-menu';
+
+const handleInputChange = event => console.log('Adjust menu items here')
+const handleSearchSubmit = event => {
+  const query = (event.target as HTMLFormElement).getElementsByTagName('input')[0].value;
+  console.log("Submitted query: ", query)
+}
+
+<SearchBar
+  autocompleteItems={[<MenuItem>Item 1</MenuItem>]}
```

Comment:
I'm guessing this will only ever be a small amount of items (no paging involved). To refine autocomplete items, type more characters. Would it ever happen where a search phase doesn't show up and scrolling is required?

Reply:
I guess it would be up to up-takers, but for example, workday's autocomplete api limits results to 7. Not sure what you mean by scrolling is required. If search phase isn't found I would guess you would either `set autocompleteItems={[]}` or to some empty state.

Reply:
If the API limits to only 7 results, I don't see a reason to support paging. Most autocomplete only show a small amount of results. I have seen UIs with unbounded results. If our UX says there should be a finite amount of results, this is perfect.